ballerina-tests/tests/06_server_streaming_client.bal

```diff
@@ -14,6 +14,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/grpc;
 // This is client implementation for server streaming scenario
```

Comment:
Can you remove this line? I have missed it when transfering the tests.

Reply:
Removed in 22fe1d0